charts/victoria-metrics-agent/values.yaml

```diff
@@ -253,6 +253,13 @@ persistence:
   # -- Bind Persistent Volume by labels. Must match all labels of targeted PV.
   matchLabels: {}
 
+# Horizontal Pod Autoscaling
+horizontalPodAutoscaler:
```

Comment:
Could you rename it to be `autoscaling`?
It will make it shorter and less error-prone to use.

Reply:
done

Reply:
@zekker6 in every other chart it is `horizontalPodAutoscaler`, if i change it to `horizontalPodAutoscaling` or just `autoscaling` then vmagent would be the outlier. Are you sure you want this to be renamed to `autoscaling`?

Reply:
@rakesh-von Good point, let's keep it as it is now then. Thanks!